pcsx2/R5900.cpp

```diff
@@ -548,6 +549,7 @@ void __fastcall eeloadHook()
 
 	wxString discelf;
 	int disctype = GetPS2ElfName(discelf);
+	psxMode = disctype == 1 ? true : false;
```

Comment:
``? true : false`` is not necessary here. (Coverity complains about such redundant operations)